app/views/kaminari/_next_page.html.haml

```diff
@@ -0,0 +1,9 @@
+-#  Link to the "Next" page
```

Comment:
Comment should have a space after the `#`<br>7 consecutive comments can be merged into one